sidebars/components.js

```diff
@@ -26,6 +26,14 @@ const getReference = (theme = 'primary') => {
     label: 'Reference',
     type: 'category',
     items: [
+      {
```

Comment:
We have a ticket in to remove this before release. This adds the demo under Reference instead of Developers > Core Contributors > Concepts